docs/_data/menu-en.yml

```diff
@@ -2,37 +2,49 @@
   - title: Overview
     buttonTitle: Overview
     subfiles:
-      - title: Introduction
+      - title: Overview
         url: /en/Overview.html
-      - title: Dora
-        url: /en/overview/Why-Dora.html
-      - title: Getting Started
-        url: /en/overview/Getting-Started.html
+
+  - title: What is Alluxio
+    buttonTitle: What_is_Alluxio
+    subfiles:
+      - title: Introduction
+        url: /en/introduction/Introduction.html
+      - title: DORA Architecture
```

Comment:
so we're styling dora as `DORA` everywhere? personally i think `dora`/`Dora` are much more prevalent and natural to read, but most importantly, i'd like the term to be consistently presented everywhere

Reply:
agree i'll change DORA to Dora but i'm deleting the architecture page from docs